library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -232,6 +232,12 @@ public int hashCode()
 
     protected ConcurrentLinkedQueue<Long> windowsForFreeMemory = new ConcurrentLinkedQueue<>();
 
+    private static boolean disableBloomFilterByDefault = false;
+    private boolean disableBloomFilter = disableBloomFilterByDefault;
```

Comment:
Different developer has different preference. It's not really impact anything like performance or complexity. 
My logic is:
if(special case)  // in this case is disabled bloom filter as treat bloom filter as default behavior
{
  do something
}

instead of 
if(not special case) 
{
  do something
}